types/dmx/dmx-tests.ts

```diff
@@ -0,0 +1,23 @@
+import DMX, { Animation } from 'dmx';
```

Comment:
```suggestion
import DMX =require('dmx');
```

and then refer to Animation as `DMX.Animation`